style.css

```diff
@@ -503,6 +521,17 @@ h1 {
     align-self: self-start;
   }
 
+  .name-group {
+    width: 100%;
```

Comment:
- [ ] Kindly 😃 try to add the background image for the footer on the mobile version as you see it is not appearing in the footer section please try to add it to make it nice like the Figma Design 💯😊 

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/ca91dab6-d06d-4c1c-98eb-84951ebf067b)|    ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/990ae080-9050-49ce-b532-38a58a8bee8b)  |